lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -583,25 +583,26 @@ def get(self, request, course_key):
             if request.GET.get('course_grade_min') or request.GET.get('course_grade_max'):
                 grade_conditions = {}
                 q_object = Q()
-                if request.GET.get('course_grade_min'):
+                course_grade_min = request.GET.get('course_grade_min')
+                if course_grade_min:
                     course_grade_min = float(request.GET.get('course_grade_min')) / 100
                     grade_conditions['percent_grade__gte'] = course_grade_min
 
-                    if course_grade_min == 0:
-                        subquery_grade_absent = ~Exists(
-                            PersistentCourseGrade.objects.filter(
-                                course_id=OuterRef('course'),
-                                user_id=OuterRef('user_id'),
-                            )
-                        )
-
-                        annotations['course_grade_absent'] = subquery_grade_absent
-                        q_object |= Q(course_grade_absent=True)
-
                 if request.GET.get('course_grade_max'):
                     course_grade_max = float(request.GET.get('course_grade_max')) / 100
                     grade_conditions['percent_grade__lte'] = course_grade_max
 
+                if not course_grade_min or course_grade_min == 0:
```

Comment:
Nit: you could actually omit the `course_grade_min == 0` check if you want, since `(not 0) == True`.  But if you like the explicitness, no harm in leaving it.

Reply:
I'm actually attempting to leverage the line `course_grade_min = float(request.GET.get('course_grade_min')) / 100` from the `if course_grade_min` condition a few lines above. Otherwise, course_grade_min would be `'0'`. If I don't do it this way, I'd have to either duplicate that code or make this condition more gnarly. Or perhaps I could do `course_grade_min== '0'`...I do realize now that this is probably pretty unclear. What do you think?

Reply:
I would probably wrap it as `float(course_grade_min)` so it's more clear on reading that we don't expect the variable to have a numeric value already, and to make explicit the cast. I think that'll also have the side effect of making more clear why the second condition is necessary.

Reply:
If the first condition is true, then `course_grade_min` will already be a float from [line 588](https://github.com/edx/edx-platform/pull/21341/files#diff-910b6d51b914745a106fb78f6bc65ffcR588). That's why I thought it might be potentially confusing.